src/main/java/com/marklogic/contentpump/MultithreadedMapper.java

```diff
@@ -203,6 +205,10 @@ public void stopRunners(int numRunners) {
                     runnerList.get(runnerList.size()-i-1).getIsShutdownDone();
             }
             if (allDone) break;
+            // Speculative fix for Bug 55693: Sleep for 0.5s between every check
+            try {
+                InternalUtilities.sleep(500);
```

Comment:
Any reason this function from mapreduce is used and not the standard thread.sleep()?